src/setupProxy.js

```diff
@@ -12,16 +12,19 @@ module.exports = (app) => {
   // );
 
   // @TRAIN
-  app.use(
-    '/api',
-    createProxyMiddleware({ target: 'https://360ApiTrain.gordon.edu/', changeOrigin: true }),
```

Comment:
The locally updated setupProxy.js file should probably not be in the pull-request.

Reply:
> 
> 
> The locally updated setupProxy.js file should probably not be in the pull-request.

Yeah...I had assumed this file would be ignored by Github 🤓

Reply:
Yes, changes to `setupProxy.js`, and to a lesser degree `.env.development` should not be pushed without good reason. `setupProxy` is important because it affects our deployments. `.env.development` only affects developers running it locally, but we should still not change the default to an API that they aren't running, that would cause myriad issues.